DuckDuckGo/AutofillLoginDetailsView.swift

```diff
@@ -261,13 +368,17 @@ private struct Copyable: ViewModifier {
     var isSelected: Bool
     var menuTitle: String
     let menuAction: () -> Void
+    let menuSecondaryTitle: String?
+    let menuSecondaryAction: (() -> Void)?
     let menuOpenedAction: () -> Void
     let menuClosedAction: () -> Void
     
-    internal init(isSelected: Bool, menuTitle: String, menuAction: @escaping () -> Void, menuOpenedAction: @escaping () -> Void, menuClosedAction: @escaping () -> Void) {
+    internal init(isSelected: Bool, menuTitle: String, menuAction: @escaping () -> Void, menuSecondaryTitle: String?, menuSecondaryAction: (() -> Void)?, menuOpenedAction: @escaping () -> Void, menuClosedAction: @escaping () -> Void) {
```

Comment:
Nit: A matter of personal preference since I don't think we have any established pattern on it, but I prefer all the closures to be at the end of the parameter list